### DISCUSSION TITLE
Add redux tag please

### DISCUSSION THREAD

**Discussion by duxrat** [dailydotdev/daily#475] [Feature Requests]
There is a lot of content about Redux, but many developers are not interested in it at all because of the available alternatives. This way I could also filter out outdated content (in news aggregator it's an issue) - I see a lot of value in adding tags of older tech. On the other side, people using Redux could subscribe to it and keep reading about it.

Or there could be a section for custom blocked words so articles with given words in the title are hidden.

**Comment by idoshamun** [comment]
Hi @yep-dev,

We're working on a new tagging system that will be able to address your concerns.
Stay tuned :)
